Gemfile

```diff
@@ -23,6 +22,8 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'csv'
```

Comment:
Ruby has a built-in CSV library. What does this gem provide in addition to that?